docs/proposals/policy-feg-api-cleanup/qos_enforcement.md

```diff
@@ -231,9 +231,8 @@ subscriber:
   # for every APN of the subscriber
   active_policies: array[string]
   # Keyed by APN IDs, and values list out the policy IDs
-  # Keys present define the APNs that the subscriber has access to
   # policy IDs define what policies will be installed for the subscriber for 
-  # each APN
+  # each APN beyond default bearers
```

Comment:
to make it clear, can you also show the existing `active_apns` field and comment that subscriber has access to these apns?